src/pages/admin/index.tsx

```diff
@@ -114,6 +121,10 @@ const AdminRouter = () => {
                 <Route path="/announcements" Component={Announcements} />
                 <Route path="/rooms" Component={Rooms} />
                 <Route path="/valentines" Component={Valentines} />
+                <Route
+                    path="/approved-valentines"
```

Comment:
TODO for someone in the future: make this /valentines/approved or something like that (two tabs on the admin panel is a bit wasteful)